noxfile.py

```diff
@@ -0,0 +1,261 @@
+"""
+Nox is a command-line tool that automates testing in multiple Python environments, similar to tox.
+Unlike tox, Nox uses a standard Python file for configuration.
+
+To run all sessions defined in the noxfile.py file, run the following command:
+$ nox
+
+To run a specific session, run the following command:
+$ nox -s <session-name>
+
+To run a session with additional arguments, run the following command:
+$ nox -s <session-name> -- <additional-arguments>
+
+To list all available sessions, run the following command:
+$ nox -l
+"""
+
+import nox
+from nox.sessions import Session
+
+# default sessions (sorted alphabetically)
+nox.options.sessions = ["bandit", "black", "flake8", "isort", "mypy", "unit-tests"]
+
```

Comment:
you should be able to set `nox.options.reuse_venv` globally as well instead of setting it with each target. I also think we'd want to set it to 'always' if possible (you can do that over CLI, not sure if it requires a boolean this way) as that is what we want I guess based on the proposal.